Update monitor-built-in-alerting.md (#13013)
pingcap/docs: tidb-cloud/monitor-built-in-alerting.md

# TiDB Cloud Built-in Alerting

TiDB Cloud provides you with an easy way to view alerts, edit alert rules, and subscribe to alert notification emails.

This document describes how to do these operations and provides the TiDB Cloud built-in alert conditions for your reference.

## View alerts

In TiDB Cloud, you can view both active and closed alerts on the Alerts page.

1. In the [TiDB Cloud console](https://tidbcloud.com/), navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page of your project.

    > **Tip:**
    >
    > If you have multiple projects, you can switch to the target project from the ☰ hover menu in the upper-left corner.

2. Click the name of the target cluster. The cluster overview page is displayed.
3. Click **Alerts** in the left navigation pane.
4. The **Alerts** page displays the active alerts by default. You can view the information of each active alert such as the alert name, trigger time, and duration. 
5. If you also want to view the closed alerts, just click the **Status** drop-down list and select **Closed** or **All**.

## Edit alert rules

> In TiDB Cloud, you can edit the alert rules by disabling or enabling the alerts or updating the alert threshold.

Reviewer: ```suggestion
In TiDB Cloud, you can edit the alert rules by disabling or enabling the alerts or updating the alert threshold.

```

# TiDB Cloud Built-in Alerting

TiDB Cloud provides you with an easy way to view alerts, edit alert rules, and subscribe to alert notification emails.

This document describes how to do these operations and provides the TiDB Cloud built-in alert conditions for your reference.

> **Note:**
>

> > Currently, the Alerts page is only available for [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters.

Reviewer: ```suggestion
> Currently, the alert feature is only available for [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters.
```

2. You can view the Alert Name, Severity, Status , Trigger time ,Duration for each closed alerts. 

## Edit Alert Rules

TiDB Cloud allows you to edit the rules for the Build-in Alerts. You can disable/enable the alerts or update the threshold. 

To edit the alert rules on the Alerts page, take the following steps:
1. In the **Alerts** page , click the "Edit Rules" button. 
2. Click Switch button to disable or enable alert rules.
3. Click the "Edit" button to update the threshold of alert rules. 

    > **Tip:**
    >
    > TiDB Cloud provide limited capability for alert rule editing . Some alert rules does not support to edit. If you would like to configure different trigger conditions, thresholds, or frequency, or have alerts automatically trigger actions in downstream services like [PagerDuty](https://www.pagerduty.com/docs/guides/datadog-integration-guide/), consider using a third-party monitoring and alerting integration. Currently, TiDB Cloud supports the [Datadog integration](/tidb-cloud/monitor-datadog-integration.md) and the [Prometheus and Grafana integration](/tidb-cloud/monitor-prometheus-and-grafana-integration.md).

> 

Author: 是的

## Subscribe to alert notification emails

> If you are a member of a project and you want to get alert notification emails of clusters in your project, take the following steps:

Reviewer: ```suggestion
To get alert notification emails of clusters in your project, take the following steps:
```

1. In the **Alerts** page , click the "Subscribe Alerts" button. 
2. Enter your email address, and then click **Subscribe**. 
    > **Tip:**
    >

>     > The alert subscription is for all the alerts in current project. If you have multiple clusters in the project, you just need to subscribe once. 

Reviewer: ```suggestion
    > The alert subscription is for all alerts in the current project. If you have multiple clusters in the project, you just need to subscribe once. 
```

> You can also add the subscription from the Alert Subscription page, take the following steps: 

Reviewer: ```suggestion
Alternatively, you can also add the subscription from the **Alert Subscription** page as follows:
```

| Condition | Recommended Action |
|:--- |:--- |

> | Total TiDB node memory utilization across cluster exceeded 70% for 10 minutes | Total TiDB node memory utilization of cluster ABC in project XYZ has exceeded 70% for 10 minutes. Consider increasing the node number or node size for TiDB to reduce the memory usage percentage of the current workload.|
> | Total TiKV node memory utilization across cluster exceeded 70% for 10 minutes | Total TiKV node memory utilization of cluster ABC in project XYZ has exceeded 70% for 10 minutes. Consider increasing the node number or node size for TiKV to reduce the memory usage percentage of the current workload. |
> | Total TiFlash node memory utilization across cluster exceeded 70% for 10 minutes | Total TiFlash node memory utilization of cluster ABC in project XYZ has exceeded 70% for 10 minutes. Consider increasing the node number or node size for TiFlash to reduce the memory usage percentage of the current workload. |
> | Total TiDB node CPU utilization exceeded 80% for 10 minutes | Total TiDB node CPU utilization of cluster ABC in project XYZ has exceeded 80% for 10 minutes. Consider increasing the node number or node size for TiDB to reduce the CPU usage percentage of the current workload.|
> | Total TiKV node CPU utilization exceeded 80% for 10 minutes | Total TiKV node CPU utilization of cluster ABC in project XYZ has exceeded 80% for 10 minutes. Consider increasing the node number or node size for TiKV to reduce the CPU usage percentage of the current workload. |
> | Total TiFlash node CPU utilization exceeded 80% for 10 minutes | Total TiFlash node CPU utilization of cluster ABC in project XYZ has exceeded 80% for 10 minutes. Consider increasing the node number or node size for TiFlash to reduce the CPU usage percentage of the current workload. |
> | TiKV storage utilization exceeds 80% | Total TiKV storage utilization of cluster ABC in project XYZ exceeds 80%. Consider increasing the node number or node storage size for TiKV to increase your storage capacity. |
> | TiFlash storage utilization exceeds 80% | Total TiFlash storage utilization of cluster ABC in project XYZ exceeds 80%. Consider increasing the node number or node storage size for TiFlash to increase your storage capacity. |

Reviewer 2: 前半句话和 condition 基本一样，有点冗余，建议删掉
```suggestion
| Total TiDB node memory utilization across cluster exceeded 70% for 10 minutes | Consider increasing the node number or node size for TiDB to reduce the memory usage percentage of the current workload.|
| Total TiKV node memory utilization across cluster exceeded 70% for 10 minutes | Consider increasing the node number or node size for TiKV to reduce the memory usage percentage of the current workload. |
| Total TiFlash node memory utilization across cluster exceeded 70% for 10 minutes | Consider increasing the node number or node size for TiFlash to reduce the memory usage percentage of the current workload. |
| Total TiDB node CPU utilization exceeded 80% for 10 minutes | Consider increasing the node number or node size for TiDB to reduce the CPU usage percentage of the current workload.|
| Total TiKV node CPU utilization exceeded 80% for 10 minutes | Consider increasing the node number or node size for TiKV to reduce the CPU usage percentage of the current workload. |
| Total TiFlash node CPU utilization exceeded 80% for 10 minutes | Consider increasing the node number or node size for TiFlash to reduce the CPU usage percentage of the current workload. |
| TiKV storage utilization exceeds 80% | Consider increasing the node number or node storage size for TiKV to increase your storage capacity. |
| TiFlash storage utilization exceeds 80% | Consider increasing the node number or node storage size for TiFlash to increase your storage capacity. |
```

Reviewer: @qqqdan PTAL
